.prettierignore

```diff
@@ -15,10 +15,3 @@ public
 *.json
 *.md
 
-# subrepos
-/packages/xsnap/moddable
-/packages/xsnap/xsnap-native
```

Comment:
How do we prevent prettier from reaching into subrepos without this?

Reply:
Indeed. This was an error introduced when I testing whether nested ignore files worked. (They don't) I'll restore. Thankfully the linter complained too!